### ISSUE TITLE
UDFs require constant focus to Excel

### ISSUE THREAD

**Issue by quantvol** [ZoomerAnalytics/xlwings#486]
I'm using xlwings 0.7.2 with Excel 2010 in Windows 7.  When I calculate a sheet with many calls to my UDF, which is rather computationally intensive, it can take several minutes for the sheet to calculate.  During this time, Excel constantly takes the focus away from any other program I try to click on or use.  This makes it impossible to do anything else on the PC while the sheet is calculating.  (To be clear, because of window focus, not any resource limitations.)  Is there a way to have Excel call the UDFs in the background without drawing focus?

**Comment by fzumstein** [comment]
yeah, when we get around to implement asynchronous functions, see bottom of #362

**Comment by briandavidgreen** [comment]
I'm not sure this is related to asynchronous functions, it sounds like quantvol is just talking about the excel screen being activated.  Somehow I have 2 boxes both with Excel 2007 on them and both running the same install of Anacondas and the same excel workbook/xlwings python file.  On one of these setups, every time the UDFs are called, the workbook is brought to the front and acquires the cursor.  On the other setup, the UDFs continue to run silently behind whatever else I'm doing on the box.  This sounds like a config issue, but I can't seem to nail it down.  Any thoughts?

Obviously the desired behavior is to NOT bring the workbook into focus on each UDF call.

**Comment by fzumstein** [comment]
These window focus issues should all be resolved with the next major release (but that's still a few months away).

**Comment by briandavidgreen** [comment]
I've confirmed that rolling back to xlwings 0.6.0 fixes this issue, FYI for anyone who's having trouble with this.

**Comment by fzumstein** [comment]
I'd assume this was fixed by the 0.9.0 rewrite
